examples/server.rs

```diff
@@ -66,11 +66,20 @@ fn main() {
     let tcp2_handle = sockets.add(tcp2_socket);
     let tcp3_handle = sockets.add(tcp3_socket);
     let tcp4_handle = sockets.add(tcp4_socket);
+    
+    let mut fragments = FragmentsSet::new(vec![]);
+    let fragment = FragmentedPacket::new(vec![0; 65535]);
+    fragments.add(fragment);
+    
 
     let mut tcp_6970_active = false;
     loop {
         let timestamp = Instant::now();
-        iface.poll(&mut sockets, timestamp).expect("poll error");
+        match iface.poll(&mut sockets, &mut fragments, timestamp) {
```

Comment:
This way the fragments are handled similarly to a socket set. Other option would be to have a fragment set attached to an interface, I am not sure which one is better.